packages/lesswrong/components/ea-forum/AdvisorsPage.tsx

```diff
@@ -0,0 +1,323 @@
+import React from 'react';
+import { Components, getSiteUrl, registerComponent } from '../../lib/vulcan-lib';
+import { AnalyticsContext, useTracking } from "../../lib/analyticsEvents";
+import { useCreate } from '../../lib/crud/withCreate';
+import { useMulti } from '../../lib/crud/withMulti';
+import { useLocation } from '../../lib/routeUtil';
+import { useCurrentUser } from '../common/withUser';
+import { useMessages } from '../common/withMessages';
+import { CopyToClipboard } from 'react-copy-to-clipboard';
+import Button from '@material-ui/core/Button';
+
+const styles = (theme: ThemeType): JssStyles => ({
+  root: {
+    background: theme.palette.grey[0],
+    padding: '24px 32px',
+  },
+  betaFlag: {
+    display: 'inline-block',
+    backgroundColor: theme.palette.background.primaryDim,
+    color: theme.palette.primary.main,
+    fontFamily: theme.typography.fontFamily,
+    fontSize: 13,
+    fontWeight: 600,
+    padding: '6px 10px',
+    borderRadius: '3px 9px 3px 3px',
+    marginTop: 8
+  },
+  headline: {
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 38,
+    fontWeight: 700,
+    marginTop: 14,
+    marginBottom: 0
+  },
+  descriptionRow: {
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 14,
+    lineHeight: '24px',
+    marginTop: 20
+  },
+  bold: {
+    fontWeight: 600,
+  },
+  italic: {
+    fontStyle: 'italic',
+  },
+  link: {
+    color: theme.palette.primary.main
+  },
+  btnsRow: {
+    display: 'flex',
+    flexWrap: 'wrap',
+    columnGap: 20,
+    rowGap: '10px',
+    marginTop: 30
+  },
+  requestBtn: {
+    textTransform: 'none',
+    fontSize: 16,
+    boxShadow: 'none',
+    padding: '14px 24px',
+  },
+  shareBtn: {
+    textTransform: 'none',
+    fontSize: 14,
+    boxShadow: 'none',
+    padding: '14px 24px',
+  },
+  requestBtnCancelText: {
+    color: theme.palette.grey[500],
+    fontFamily: theme.typography.fontFamily,
+    marginTop: 10
+  },
+  advisorsHeadline: {
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 22,
+    fontWeight: 700,
+    marginTop: 30,
+  },
+  advisors: {
+    display: 'grid',
+    gridTemplateColumns: 'repeat(auto-fill, minmax(212px, 1fr))',
+    padding: '30px 10px',
+    columnGap: 20,
+    rowGap: '30px',
+    borderTop: `2px solid ${theme.palette.primary.main}`,
+    borderBottom: `2px solid ${theme.palette.primary.main}`,
+    marginTop: 10,
+  },
+  feedbackText: {
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 16,
+    marginTop: 40
+  },
+  contactEmail: {
+    color: theme.palette.primary.main,
+    fontFamily: theme.typography.postStyle.fontFamily,
+    fontSize: 16,
+    marginTop: 6
+  }
+})
+
+const advisors = [
```

Comment:
I believe there will be at least one more advisor to add before we launch tomorrow.